osc-domain/src/main/java/org/osc/core/broker/model/entities/virtualization/SecurityGroup.java

```diff
@@ -75,7 +75,7 @@
 
     @OneToMany(mappedBy = "securityGroup", fetch = FetchType.LAZY, cascade = CascadeType.ALL)
     @OrderBy("type")
-    private Set<SecurityGroupMember> securityGroupMembers = new TreeSet<>();
+    private Set<SecurityGroupMember> securityGroupMembers = new HashSet<>();
```

Comment:
one of the reasons why we used a tree set was to have a sorted list of members. Please check to see if any code relies on this collection being sorted.

Also since we are not putting the SecurityGroupMember in a tree set....it does not need to implement comparable interface. Please update the entity as required.